### ISSUE TITLE
Ladder Man (1992)

### ISSUE THREAD

**Issue by Python-Exoproject** [exoscoriae/eXoDOS#1164]
Episode two is an empty folder in the zip and run.bat has episode two noted as missing. Ladder Man episodes 1, 2, & 3 are all part of the game "Ladder Man (1992)"

**Comment by exoscoriae** [comment]
this is correct. just updated ladder man 1 and 3 to registered versions. Seems even the author doesn't have a copy of ladder man 2. but I'll leave this open in case it ever shows up.
